From 3481141e1c08f9a129ed0bdcd1ec342dbbbbeb8f Mon Sep 17 00:00:00 2001
From: Steve Kondik <steve@cyngn.com>
Date: Sat, 8 Nov 2014 06:16:24 -0800
Subject: [PATCH 07/29] skia: Update QC-PERF board flag to TARGET_HAVE_QC_PERF

  * Some targets will set QCOM_HARDWARE but won't have these libs.

Change-Id: I04575a14327edd0985bef71a376f8cb998740fa5
Signed-off-by: Chet Kener <Cl3Kener@gmail.com>
---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index dc2863f..47831cc 100644
--- a/Android.mk
+++ b/Android.mk
@@ -531,7 +531,7 @@ LOCAL_STATIC_LIBRARIES := \
 	libgif \
 	libsfntly
 
-ifeq ($(BOARD_USES_QCOM_HARDWARE),true)
+ifeq ($(TARGET_HAVE_QC_PERF),true)
 	LOCAL_WHOLE_STATIC_LIBRARIES += libqc-skia
 endif
 
-- 
2.5.0

